survey-sdk/src/test/java/co/nimblehq/survey/serializer/DataTest.kt

```diff
@@ -0,0 +1,167 @@
+    "          {\n" +
+    "            \"id\": \"d3afbcf2b1d60af845dc\",\n" +
+    "            \"type\": \"question\"\n" +
+    "          },\n" +
+    "          {\n" +
+    "            \"id\": \"940d229e4cd87cd1e202\",\n" +
+    "            \"type\": \"question\"\n" +
+    "          }\n" +
+    "        ]\n" +
+    "      }\n" +
+    "    }\n" +
+    "  },\n" +
+    "  \"included\": [\n" +
+    "    {\n" +
+    "      \"id\": \"d3afbcf2b1d60af845dc\",\n" +
+    "      \"type\": \"question\",\n" +
+    "      \"attributes\": {\n" +
+    "        \"text\": \"\\nThank you for visiting Scarlett!\\n Please take a moment to share your feedback.\",\n" +
+    "        \"help_text\": null,\n" +
+    "        \"display_order\": 0,\n" +
+    "        \"short_text\": \"introduction\",\n" +
+    "        \"pick\": \"none\",\n" +
+    "        \"display_type\": \"intro\",\n" +
+    "        \"is_mandatory\": false,\n" +
+    "        \"correct_answer_id\": null,\n" +
+    "        \"facebook_profile\": null,\n" +
+    "        \"twitter_profile\": null,\n" +
+    "        \"image_url\": \"https://dhdbhh0jsld0o.cloudfront.net/m/2001ebbfdcbf6c00c757_\",\n" +
+    "        \"cover_image_url\": \"https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_\",\n" +
+    "        \"cover_image_opacity\": 0.6,\n" +
+    "        \"cover_background_color\": null,\n" +
+    "        \"is_shareable_on_facebook\": false,\n" +
+    "        \"is_shareable_on_twitter\": false,\n" +
+    "        \"font_face\": null,\n" +
+    "        \"font_size\": null,\n" +
+    "        \"tag_list\": \"\"\n" +
+    "      },\n" +
+    "      \"relationships\": {\n" +
+    "        \"answers\": {\n" +
+    "          \"data\": []\n" +
+    "        }\n" +
+    "      }\n" +
+    "    },\n" +
+    "    {\n" +
+    "      \"id\": \"575db8c074601994bde3\",\n" +
+    "      \"type\": \"answer\",\n" +
+    "      \"attributes\": {\n" +
+    "        \"text\": \"Email \",\n" +
+    "        \"help_text\": null,\n" +
+    "        \"input_mask_placeholder\": \"you@example.com\",\n" +
+    "        \"short_text\": \"answer_6\",\n" +
+    "        \"is_mandatory\": false,\n" +
+    "        \"is_customer_first_name\": false,\n" +
+    "        \"is_customer_last_name\": false,\n" +
+    "        \"is_customer_title\": false,\n" +
+    "        \"is_customer_email\": true,\n" +
+    "        \"prompt_custom_answer\": false,\n" +
+    "        \"weight\": null,\n" +
+    "        \"display_order\": 2,\n" +
+    "        \"display_type\": \"default\",\n" +
+    "        \"input_mask\": \"[\\\\.\\\\-_\\\\+a-zA-Z0-9]+@[\\\\-\\\\a-zA-Z0-9]+(?:\\\\.[\\\\-a-zA-Z0-9]+)+\",\n" +
+    "        \"date_constraint\": null,\n" +
+    "        \"default_value\": null,\n" +
+    "        \"response_class\": \"string\",\n" +
+    "        \"reference_identifier\": null,\n" +
+    "        \"score\": null,\n" +
+    "        \"alerts\": []\n" +
+    "      }\n" +
+    "    }\n" +
+    "  ]\n" +
+    "}"
```

Comment:
I would really suggest using json resources for those test data. We can do this improvement in the next phase

Reply:
@luongvo Yes , we can do this as improvement next phase.